package.json

```diff
@@ -69,6 +69,7 @@
     "gh-pages": "2.0.1",
     "husky": "1.3.1",
     "jest": "24.8.0",
+    "jest-styled-components": "6.3.3",
```

Comment:
🤔 I already committed it normally

Reply:
It's not up to date, on it ^^

Reply:
Can you remove my previous `jest-styled-components` import in the deps ? Thanks :)

Reply:
Done